Ashamokwu_form.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width">
+  <title>Intro to HTML and Javascript</title>
+  <link href="style.css" rel="stylesheet" type="text/css" />
+</head>
+
+<body>
+    <form>
+        <label>First Name:</label><br>
+        <input type="text" id="First Name" Name="Firs Name" ><br>
```

Comment:
There is an error in your name. Also, names and id's should not include any spaces,